tests/h/views/activity_test.py

```diff
@@ -594,25 +594,101 @@ def test_search_passes_the_display_name_to_the_template_if_the_user_has_one(
 
         assert username == user.display_name
 
-    def test_search_passes_the_annotation_count_to_the_template(self,
-                                                                controller,
-                                                                search,
-                                                                user,
-                                                                user_service):
+    def test_search_passes_the_public_annotation_count_to_the_template(self,
+                                                                       controller,
+                                                                       search,
+                                                                       user,
+                                                                       user_service):
         controller.search()
 
-        user_service.annotation_count.assert_called_once_with(user.userid)
+        user_service.public_annotation_count.assert_called_once_with(user.userid)
 
-    def test_search_does_not_pass_the_annotation_count_to_the_template_when_flag_is_disabled(self,
-                                                                                             controller,
-                                                                                             search,
-                                                                                             user,
-                                                                                             user_service,
-                                                                                             pyramid_request):
+    def test_search_does_not_pass_the_public_annotation_count_to_the_template_when_flag_is_disabled(self,
+                                                                                                    controller,
+                                                                                                    search,
+                                                                                                    user,
+                                                                                                    user_service,
+                                                                                                    pyramid_request):
         pyramid_request.feature.flags['total_shared_annotations'] = False
         controller.search()
 
-        assert user_service.annotation_count.called == False
+        assert user_service.public_annotation_count.called is False
+
+    def test_search_does_not_pass_the_private_annotation_count_to_the_template_when_user_is_different_from_authenticated_user(self,
```

Comment:
Some of these test names are getting pretty silly :joy:

It's ok for the test names to be a bit shorter, and then add a docstring to the test method explaining to the reader what it's supposed to be testing.